src/sources/darwin/openWifiConnections.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env kmd
+exec defaults read /Library/Preferences/SystemConfiguration/com.apple.airport.preferences
+extract .*Open.*
```

Comment:
I think we should only need one extract, something like this should work
`SecurityType=Open`